server/container/cli/src/main/java/org/apache/james/cli/ServerCmd.java

```diff
@@ -44,7 +44,6 @@
 import org.apache.james.cli.probe.impl.JmxQuotaProbe;
 import org.apache.james.cli.probe.impl.JmxSieveProbe;
 import org.apache.james.cli.type.CmdType;
-import org.apache.james.cli.utils.ValueWithUnit;
```

Comment:
I don't agree with the whole commit : you didn't generify, I did exactly the opposite.
ValueWithUnit could be used for time for example or weight, now by renaming it as SizeWithUnit, it's only size.
BTW, SizeWithUnit looks a bit dumb, a size always has a unit otherwise it's just a number, right ?
Finally, if you want a Size type, just create one on top of ValueWithUnit.

Reply:
Here generify is 'moving it out of cli so that other part of James (conf loading?) can use it'.

The ValueWithUnit name **is a lie** as unit convertion targets sizes. (IMO we don't care about this genericity).

Finally, yes it's a bit dumb, but do we want to solve it here? That will be time consuming.

Reply:
If we always use it as a Size, than ok to rename it as Size (not SizeWithUnit)